### PR TITLE
chore: skip integration tests in short mode

### LINKED ISSUE #6483: Failing operator tests on daily release
## This is a...

<pre><code>
[ ] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Documentation issue or request
</code></pre>

## Description

The operator tests are failing on the [daily release](https://ci.fabric8.io/job/syndesis-release-daily/567/console) with:

```
07:16:58 [91m[0m?   	github.com/syndesisio/syndesis/install/operator/cmd/manager	[no test files]
07:18:03 ?   	github.com/syndesisio/syndesis/install/operator/pkg	[no test files]
07:18:03 ?   	github.com/syndesisio/syndesis/install/operator/pkg/apis	[no test files]
07:18:03 ?   	github.com/syndesisio/syndesis/install/operator/pkg/apis/syndesis/v1alpha1	[no test files]
07:18:03 ?   	github.com/syndesisio/syndesis/install/operator/pkg/build	[no test files]
07:18:03 ?   	github.com/syndesisio/syndesis/install/operator/pkg/cmd	[no test files]
07:18:03 ?   	github.com/syndesisio/syndesis/install/operator/pkg/cmd/internal	[no test files]
07:18:06 ok  	github.com/syndesisio/syndesis/install/operator/pkg/cmd/internal/grant	0.077s
07:18:06 operator was installed successfully
07:18:06 error: could not locate a kubeconfig
07:18:06 FAIL	github.com/syndesisio/syndesis/install/operator/pkg/cmd/internal/install	0.044s
07:18:06 ?   	github.com/syndesisio/syndesis/install/operator/pkg/cmd/internal/run	[no test files]
07:18:10 --- FAIL: TestUninstall (0.00s)
07:18:10     uninstall_test.go:54: 	Test: When run without any CRs, `operator uninstall` should not fail
07:18:10     uninstall_test.go:56: 	✗	 got an error when uninstalling the app: [could not locate a kubeconfig]
07:18:10 FAIL
07:18:10 FAIL	github.com/syndesisio/syndesis/install/operator/pkg/cmd/internal/uninstall	0.042s
07:18:10 ?   	github.com/syndesisio/syndesis/install/operator/pkg/controller	[no test files]
07:18:10 ?   	github.com/syndesisio/syndesis/install/operator/pkg/controller/syndesis	[no test files]
07:18:10 ok  	github.com/syndesisio/syndesis/install/operator/pkg/generator	0.058s
07:18:11 ?   	github.com/syndesisio/syndesis/install/operator/pkg/generator/assets	[no test files]
07:18:11 ?   	github.com/syndesisio/syndesis/install/operator/pkg/generator/dev	[no test files]
07:18:11 ?   	github.com/syndesisio/syndesis/install/operator/pkg/openshift	[no test files]
07:18:11 ?   	github.com/syndesisio/syndesis/install/operator/pkg/openshift/serviceaccount	[no test files]
07:18:11 ?   	github.com/syndesisio/syndesis/install/operator/pkg/openshift/template	[no test files]
07:18:14 ok  	github.com/syndesisio/syndesis/install/operator/pkg/syndesis/action	0.042s
07:18:14 ok  	github.com/syndesisio/syndesis/install/operator/pkg/syndesis/configuration	0.035s
07:18:14 ?   	github.com/syndesisio/syndesis/install/operator/pkg/syndesis/legacy	[no test files]
07:18:14 ?   	github.com/syndesisio/syndesis/install/operator/pkg/syndesis/operation	[no test files]
07:18:14 ?   	github.com/syndesisio/syndesis/install/operator/pkg/syndesis/template	[no test files]
07:18:16 ok  	github.com/syndesisio/syndesis/install/operator/pkg/util	0.020s
07:18:17 The command '/bin/sh -c go test -mod=vendor ./cmd/... ./pkg/...' returned a non-zero code: 1
```

### PR DESCRIPTION
**Pull request by zregvart**
By default now we only run short tests. Full test needs to be run manually by running `go test -mod=vendor ./cmd/... ./pkg/...`. (Changes in a1560f5)

Also fixes formatting and adds uncommitted vendored files.

Fixes #6483

### CODE REVIEW

File: install/operator/pkg/cmd/internal/grant/grant_test.go [hunk 1]
```diff
@@ -17,130 +17,130 @@
...
-        } else {
-            t.Logf("\t%s\t the rolebinding only have one subject associated", succeed)
-
-            if rb.Subjects[0].Name != user {
-                t.Fatalf("\t%s\t the rolebinding's associated subject should be [%s], but got [%s]", failed, user, rb.Subjects[0].Name)
-            }
-            t.Logf("\t%s\t the rolebinding's associated subject is [%s]", succeed, user)
-        }
-    }
+	if os.Getenv("CI") != "" {
```
Comment (lgarciaaco) [inline install/operator/pkg/cmd/internal/grant/grant_test.go]:
```suggestion
		if testing.Short() {
```

File: install/operator/pkg/cmd/internal/grant/grant_test.go [hunk 2]
```diff
@@ -17,130 +17,130 @@
...
-        } else {
-            t.Logf("\t%s\t the clusterrolebinding only have one subject associated", succeed)
-
-            if crb.Subjects[0].Name != user {
-                t.Fatalf("\t%s\t the clusterrolebinding's associated subject should be [%s], but got [%s]", failed, user, crb.Subjects[0].Name)
-            }
-            t.Logf("\t%s\t the clusterrolebinding's associated subject is [%s]", succeed, user)
-        }
-    }
+	if os.Getenv("CI") != "" {
```
Comment (lgarciaaco) [inline install/operator/pkg/cmd/internal/grant/grant_test.go]:
```suggestion
	if testing.Short() {
```

**Comment by zregvart** [comment]
@lgarciaaco missed those, thanks, I'll update in a bit.

**Comment by zregvart** [comment]
@lgarciaaco I've added 86accc301966a163a3ca3d72a8a15d58a50760f8 those tests look like unit tests (they work offline) and execute fast, so I thought best not skip them -- mark them as non-short.

**Comment by lgarciaaco** [comment]
> @lgarciaaco I've added [86accc3](https://github.com/syndesisio/syndesis/commit/86accc301966a163a3ca3d72a8a15d58a50760f8) those tests look like unit tests (they work offline) and execute fast, so I thought best not skip them -- mark them as non-short.

@zregvart they will fails because down the code the `config` is still needed and the test fails. Leave them out for now I am working in a permanent solution so we can also include them

**Comment by lgarciaaco** [comment]
@zregvart you right, `grant_test.go` will pass

**Comment by zregvart** [comment]
I've rebased on top of #6489 to make the CI check pass.

**Comment by lgarciaaco** [comment]
@zregvart done?? :)

**Comment by zregvart** [comment]
I provoked a change on the PR by setting the `group/operator` label, this made the pure bot merge it :)

**Pull request merged**